deps.edn

```diff
@@ -32,15 +32,20 @@
         com.fzakaria/slf4j-timbre   {:mvn/version "0.4.1"}
         org.slf4j/log4j-over-slf4j  {:mvn/version "2.0.13"}
         org.slf4j/jul-to-slf4j      {:mvn/version "2.0.13"}
-        org.slf4j/jcl-over-slf4j    {:mvn/version "2.0.13"}}
+        org.slf4j/jcl-over-slf4j    {:mvn/version "2.0.13"}
+        org.scicloj/kindly-advice   {:mvn/version "1-beta12"}
+        org.scicloj/kindly-render   {:mvn/version "0.1.1-alpha"}
```

Comment:
Are there non-alpha/beta versions we could use instead?

Reply:
not yet.
I keep a close eye on this, and maybe @daslu might agree to make non beta releases soon. (at least of "org.scicloj/kindly")


What is important to note: From an end user of clojupyter perspective , "nothing will change".
The "existing kinds" which are in library "org.scicloj/kindly {:mvn/version "4-beta14"}" are stable.
(new ones might be added over time)
The code which "renders kinds" is still a bit in flux.
(parts of it is in the PR, parts of it in some of the support libraries)
But we can "change this" while we go, and the enduser of clojupyter will not notice.